app/src/components/buttons/toggleButton/toggleButton.scss

```diff
@@ -53,3 +52,35 @@
     background: $COLOR--gray-91;
   }
 }
+
+.separated {
+  display: flex;
+  height: 23px;
+
+  .button-item {
+    display: inline-block;
+    float: none;
+    height: 100%;
+    border: none;
+    border-radius: 100px;
+    margin: 0 6px;
+    padding: 5px 15px;
+    color: $COLOR--gray-47;
+    background-color: $COLOR--white-four;
+    font-family: $FONT-SEMIBOLD;
+    font-size: 12px;
+    opacity: 0.5;
+
+    &.active {
+      color: $COLOR--black-2;
+      background-color: $COLOR--white-two;
+      box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.2);
+      opacity: 1;
+      cursor: default;
+    }
+
+    span {
+      display: inline;
```

Comment:
But `span` tag inline by default.

Reply:
It should replace the previous definition of the display property.
![image](https://user-images.githubusercontent.com/31861805/49582454-222aa280-f966-11e8-96b5-df598212e2eb.png)